### ISSUE TITLE
Update static stac-browsers

### ISSUE THREAD

**Issue by anayeaye** [NASA-IMPACT/veda-backend#124]
# What
We have upgraded our veda-backend databases and APIs but have not updated all of the static stac browsers to use the latest APIs. These should be updated and additional browsers should be deployed as needed to cover all supported stages.

- staging browser is directed to old v0.4.5 stack http://delta-staging-stac-browser.s3-website-us-east-1.amazonaws.com/
- dev browser is pointed at a non-existent stac-api http://delta-dev-stac-browser.s3-website-us-east-1.amazonaws.com/

# AC
- [x] A dedicated stac-browser is deployed for each stage of the veda-backend stac-api
- [x] The VEDA Documentation site is updated as needed

**Comment by smohiudd** [comment]
Here is a new endpoint for the new dev browser: http://veda-dev-stac-browser.s3-website-us-west-2.amazonaws.com/

For documentation, I believe this is the only place that references the old endpoint.

**Comment by smohiudd** [comment]
@anayeaye, the STAC apis catalog titles for dev and staging probably should updated. Dev uses `veda-stac` and staging uses `delta-backend-stac`

**Comment by anayeaye** [comment]
@smohiudd good catch. We fixed that naming in[ this PR](https://github.com/NASA-IMPACT/veda-backend/pull/134/files#diff-e4855afb79837f4126dcf1dd5f9ae77c2c6c49059e75a5151bf6949881afefecL52) but because we haven't merged develop into main we don't have that change deployed (develop is tied to dev stack deployment, main to staging). 

I think we should open a PR to merge develop into main to redeploy the staging stack to update delta to veda in the swagger docs. Maybe we can set a reminder to do that first thing next year? I'm hesitant to make any changes at this point. I also just opened a [related PR](https://github.com/NASA-IMPACT/veda-backend/issues/138) related to github deployment actions that shouldn't be needed for the renaming changes but is worth keeping in mind.
